docs/fundamentals/code-analysis/style-rules/language-rules.md

```diff
@@ -71,202 +71,6 @@ The style rules in this section are applicable to both C# and Visual Basic.
 - [Parameter preferences](#parameter-preferences)
   - dotnet\_code\_quality\_unused\_parameters
 
-### <a name="this-and-me"></a>"This." and "Me." qualifiers
```

Comment:
The actual rule contents moved out to separate docs which can directly be linked from the analyzer by just mapping the rule ID to URI.